sdk/python/kfp/gcp.py

```diff
@@ -14,7 +14,8 @@
 
 from kubernetes.client import V1Toleration
 
-def use_gcp_secret(secret_name='user-gcp-sa', secret_file_path_in_volume='/user-gcp-sa.json', volume_name='gcp-credentials', secret_volume_mount_path='/secret/gcp-credentials'):
+
+def use_gcp_secret(secret_name='user-gcp-sa', secret_file_path_in_volume='/user-gcp-sa.json', volume_name=None, secret_volume_mount_path='/secret/gcp-credentials'):
```

Comment:
why not switch the order of volume_name and secret_volume_mount_path?

Reply:
It would look a tiny bit better, but parameter re-ordering is a breaking change signature-wise.
If someone calls it using positional arguments, their code will break.